buildenvs/xen.Dockerfile

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022, Unikraft GmbH and The KraftKit Authors.
+# Licensed under the BSD-3-Clause License (the "License").
+# You may not use this file except in compliance with the License.
+
+FROM kraftkit.sh/base-golang:latest
```

Comment:
shouldnt this be based on `base`? just asking

Reply:
`base-golang` is the same as `base` with Go installed.